### PR TITLE
*: refactor executor.checkHandleExists to tables.CheckHandleExists to reuse code

### PR DESCRIPTION
**Pull request by winkyao**
ref #5138

### CODE REVIEW

**Comment by winkyao** [comment]
@zimulala PTAL

File: table/tables/tables.go [hunk 1]
```diff
@@ -810,6 +802,24 @@ func FindIndexByColName(t table.Table, name string) table.Index {
 	return nil
 }
 
+// CheckHandleExists Check if recordID key exists. if not exists, return nil,
```
Comment (zimulala) [inline table/tables/tables.go]: s/Check/checks
s/if/If

**Comment by zimulala** [comment]
Rest LGTM

**Comment by winkyao** [comment]
@zimulala @coocood @shenli PTAL

**Pull request merged**